framework/include/base/Factory.h

```diff
@@ -136,7 +148,8 @@ class Factory
   template<typename T>
   void reg(const std::string & obj_name)
   {
-    /**
+
+    /*
```

Comment:
Looks like you dropped an asterisk here, I think that breaks the Doxygen for this function.